### PR TITLE
zh-translation: Update index.md#6234

### PR DESCRIPTION
**Pull request by owen-sun**
标题：zh-translation: https://github.com/istio/istio.io/blob/master/content/en/about/release-cadence/index.md
内容：
ref:#1701
[X]  About

### CODE REVIEW

**Comment by owen-sun** [comment]
@googlebot I signed it!

**Comment by owen-sun** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> #### What to do if you already signed the CLA
> ##### Individual signers
> 
>     * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> 
> ##### Corporate signers
> 
>     * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> 
>     * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
>     * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fistio%2Fistio.io%2Fpull%2F6234) for more info**.



> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> #### What to do if you already signed the CLA
> ##### Individual signers
> 
>     * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> 
> ##### Corporate signers
> 
>     * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> 
>     * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
>     * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fistio%2Fistio.io%2Fpull%2F6234) for more info**.

@googlebot I signed it!

**Comment by owen-sun** [comment]
@gorda please review this PR.

File: content/zh/about/release-cadence/index.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ---
-title: 构建 & 发布节奏
+title: 构建 & 发布节奏 {#build-release-cadence}
```
Comment (gorda) [inline content/zh/about/release-cadence/index.md]: > title: 构建 & 发布节奏 {#build-release-cadence}

建议：title: 构建 & 发布节奏
title 不加锚点

File: content/zh/about/release-cadence/index.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-|LTS 版本         | 在下一个 LTS 之后的 3 个月内提供支持                     | 安全地部署在生产中。建议用户尽快升级到这些版本。
-|补丁             | 与相应的 Snapshot/LTS 版本相同                           | 鼓励用户在给定版本可用时立即采用补丁版本。
+|开发构建         | 不支持                                                   | 不完全可靠。建议用于实验、测试场景。
+|LTS 版本         | 在下一个 LTS 版本后的 3 个月内提供支持                     | 可安全地用于生产环境。建议用户尽快升级到这些版本。
+|补丁             | 与相应的 Snapshot/LTS 版本相同                           | 建议用户在补丁可用时尽快采用。
 
-您可以在[发布页面](https://github.com/istio/istio/releases)上找到可用的版本，如果您是冒险类型，您可以在[开发构建 wiki](https://github.com/istio/istio/wiki/Dev%20Builds) 上了解我们的开发构建。您可以在[此处](/zh/news)找到每个 LTS 版本的高级发行说明。
+您可以在[发布页面](https://github.com/istio/istio/releases)上找到可用的版本，如果您想了解、体验最新特性，您可以在[开发构建 wiki](https://github.com/istio/istio/wiki/Dev%20Builds) 上了解我们的开发构建版本。您可以在[此处](/zh/news)找到每个 LTS 版本的简要发行说明。
 
 ## 命名模式
```
Comment (gorda) [inline content/zh/about/release-cadence/index.md]: > ## 命名模式

建议：## 命名模式{#naming-scheme}

现在的文档，二级/三级标题都需要手动添加锚点

File: content/zh/about/release-cadence/index.md [hunk 3]
```diff
@@ -25,12 +25,12 @@ LTS 版本的命名模式为：
...
 
-其中`<minor>`针对每个 LTS 版本增加，`<LTS patch level>`为当前 LTS 版本的补丁总数。补丁通常是相对于 LTS 的小变更。
+其中`<minor>`针对每个 LTS 版本递增，`<LTS patch level>`为当前 LTS 版本补丁总数。补丁相对于 LTS 通常是较小变更。
 
 对于 snapshot 版本，命名模式为：
 
-{{< text plain >}}
-<major>.<minor>.0-snapshot.<snapshot count>
+{{< text plain >}} 
+<major>.<minor>.<sha>
```
Comment (gorda) [inline content/zh/about/release-cadence/index.md]:
```
{{< text plain >}} 
<major>.<minor>.<sha> 
```

英文版的 [master](https://github.com/istio/istio.io/blob/master/content/en/about/release-cadence/index.md) 内容似乎是这样的：
```
{{< text plain >}}
<major>.<minor>-alpha.<sha>
```

File: content/zh/about/release-cadence/index.md [hunk 4]
```diff
@@ -25,12 +25,12 @@ LTS 版本的命名模式为：
...
 对于 snapshot 版本，命名模式为：
 
-{{< text plain >}}
-<major>.<minor>.0-snapshot.<snapshot count>
+{{< text plain >}} 
+<major>.<minor>.<sha> 
 {{< /text >}}
 
-其中`<major>.<minor>`代表下一个LTS，`<sha>`代表git提交版本的构建发布。
+其中`<major>.<minor>`代表下一个LTS，`<sha>`代表此版本构建基于的git提交。
```
Comment (gorda) [inline content/zh/about/release-cadence/index.md]: > 其中`<major>.<minor>`代表下一个LTS，`<sha>`代表此版本构建基于的git提交。

建议：其中 `<major>.<minor>` 代表下一个 LTS，`<sha>` 代表此版本构建基于的 git 提交。

File: content/zh/about/release-cadence/index.md (hunk 3 above)
Comment (owen-sun) [reply content/zh/about/release-cadence/index.md]: 这个地方我有个疑问，issue上我也@了SataQiu。我看到的英文原版是这个text(https://github.com/istio/istio.io/blob/master/content/en/about/release-cadence/index.md)，这里没有<sha>呢？

For snapshot releases, our naming scheme is:

{{< text plain >}} .-alpha. {{< /text >}}

where <major>.<minor> represent the next LTS, and <sha> represents the git commit the release is built from.

File: content/zh/about/release-cadence/index.md (hunk 4 above)
Comment (owen-sun) [reply content/zh/about/release-cadence/index.md]: @gorda 抱歉啊没看出来你建议的和原来翻译的差别？

**Comment by owen-sun** [comment]
@gorda 按照您的comments修改了一般，帮忙再review下哈，多谢!

File: content/zh/about/release-cadence/index.md [hunk 5]
```diff
@@ -5,32 +5,32 @@ weight: 15
...
 {{< /text >}}
 
-其中`<minor>`针对每个 LTS 版本增加，`<LTS patch level>`为当前 LTS 版本的补丁总数。补丁通常是相对于 LTS 的小变更。
+其中 `<minor>` 针对每个 LTS 版本递增，`<LTS patch level>`为当前 LTS 版本补丁总数。补丁相对于 LTS 通常是较小变更。
 
 对于 snapshot 版本，命名模式为：
 
-{{< text plain >}}
-<major>.<minor>.0-snapshot.<snapshot count>
+{{< text plain >}}
```
Comment (gorda) [inline content/zh/about/release-cadence/index.md]: `{{< text plain >}} `

lint 报错了，需要删除末尾的空格

**Pull request merged**